packages/flutter_driver/lib/src/driver.dart

```diff
@@ -15,13 +15,37 @@ import 'health.dart';
 import 'message.dart';
 import 'timeline.dart';
 
+enum TracingCategory {
```

Comment:
An `enum`! Even better. Where did you get the list? Maybe add a doc comment providing the link to this list? Could be useful in the future if the list ever changes.